test/lambda_toolshed/papillon/channel_test.cljc

```diff
@@ -0,0 +1,69 @@
+(ns lambda-toolshed.papillon.channel-test
+  "Tests that demonstrate the opt-in ability of papillon to work seamlessly with
+  clojure.core.async channels -either as the deferred output of an interceptor
+  or as the output of the overall chain execution."
+  (:require
```

Comment:
looks like there's some variation going on in the repo right now with the `:require`'s in the `ns` forms. there's several using the older style (first vector on the same line), and only a couple using the newer style like here.

is it ok to just stick with the older style?

Reply:
I'm still stuck on the older style, but I'm OK with either.  @stevenproctor , do you have a preference?

Reply:
granted, unless there was a particular reason for the delta that I don't remember at the moment.